docs/workflow.md

```diff
@@ -0,0 +1,137 @@
+
+# Workflows
+
+Ths document documents two different yet common workflows in BKR using `kubeprod` and `kubecfg`:
+
+* Basic workflow
+* [Advanced workflow](#advanced-workflow)
+
+## Pre-requisites
+
+* An existing Kubernetes cluster.
+* `kubeprod` binary from https://github.com/bitnami/kube-prod-runtime/releases
+
+## Advanced workflow
+
+1. Ensure the default `kubectl` context
+1. Install BKPR into an existing Kubernetes cluster
+1. Implement customizations (optional)
+1. Test changes (only if BKPR is already deployed)
+1. Deploy changes
+1. Check-in changes
+
+### Ensure the default `kubectl` context
+
+Ensure your default context in the Kubernetes `kubectl` client is the expected one:
+
+```
+$ kubectl config current-context
+gke_bkprtesting_europe-west1-b_felipe
+```
+
+and
+
+```
+$ kubectl cluster-info
+Kubernetes master is running at https://35.240.18.142
+GLBCDefaultBackend is running at https://35.240.18.142/api/v1/namespaces/kube-system/services/default-http-backend:http/proxy
+Heapster is running at https://35.240.18.142/api/v1/namespaces/kube-system/services/heapster/proxy
+KubeDNS is running at https://35.240.18.142/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+Metrics-server is running at https://35.240.18.142/api/v1/namespaces/kube-system/services/https:metrics-server:/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+```
+
+### Deploy BKPR into the Kubernetes cluster
```

Comment:
In this section you are not deploying BKPR as with --only-generate you are generating the cloud config. I would change the name of this section